RefactorTest/RenameCpp20.cpp

```diff
@@ -456,6 +456,44 @@ void f10()
     }
 }
 
+// using for scoped enums
+void f11()
+{
+    //#GOINK#: ID Rename E
+    enum class E : uint8_t
```

Comment:
This fails to build when the enum is local to this function.  I suspect that is because the anonymous class created for the lambda is emitted to the enclosing namespace scope and not as a local inside this function.  If I move the enum to namespace scope, then it compiles.  Please update your code to put the enum in the namespace scope to correct this build error.

Reply:
Also you haven't included a header to define `uint8_t` and it's not important to this test, so just drop off the `: uint8_t` base type and declare `enum class E`